Mail/Views/SplitView.swift

```diff
@@ -231,7 +231,7 @@ struct SplitView: View {
             return false
         }
 
-        return appLaunchCounter.value > Constants.minimumOpeningBeforeSync
+        return true
```

Comment:
Maybe we can simplify the logic? Wh should show the view if `UserDefaults.shared.shouldPresentSyncDiscovery && !appLaunchCounter.isFirstLaunch`?
We might even be able to remove `appLaunchCounter.isFirstLaunch` as we want to present this bottom sheet asap